content/blog/2020-11-16-BlackInDataWeek/index.Rmarkdown

```diff
@@ -0,0 +1,115 @@
+---
+title: '#BlackInDataWeek 2020'
+author:
+  - alison
+  - greg
+date: '2020-11-16'
+categories:
+  - learn
+description: |
+  Learn, celebrate, and amplify Black voices in informatics, data science, and coding around the world.
+slug: blackindataweek
+photo:
+  author: Luana Azevedo
+  url: https://unsplash.com/photos/AHBNGvRTm_A
+---
+
+```{r setup, include=FALSE}
+library(tidyverse)
+library(glue)
+library(gt)
+```
+
+Today marks the first day of a week-long celebration called [`#BlackinDataWeek`](https://blkindata.github.io/). It is a week to amplify Black voices in `#informatics`, `#datascience`, & `#coding` around the world. The goals are to:
```

Comment:
"It is a week" <- "Its aim is" (to avoid repeating the word "week")

Reply:
"&" <- "and"

Reply:
Just realized that if we change the sentence to "Its aim is to" we'll need to change the opening verbs below to "providing", "encouraging", and "elevating".